modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ScalaAkkaClientCodegen.java

```diff
@@ -116,6 +116,7 @@ public ScalaAkkaClientCodegen() {
         importMapping.remove("List");
         importMapping.remove("Set");
         importMapping.remove("Map");
+        importMapping.put("BigDecimal", "java.math.BigDecimal");
```

Comment:
Scala has it's own `scala.math.BigDecimal` type and global alias `BigDecimal`
https://www.scala-lang.org/api/2.12.0/scala/math/BigDecimal.html
Should we use java type? As those types can be interfered in the client code. Then I will have to write something like this:
```scala
import java.math.{ BigDecimal => JBigDecimal }
...
 val income:  BigDecimal  = db.getIncome(..)
 val outcome: JBigDecimal = generatedClient.getOutcome(..)
 val balance: BigDecimal  = income - BigDecimal(outcome)
```

Reply:
We can use `scala.math.BigDecimal` if that's preferable.

I used `java.math.BigDecimal` in this change as it's copied from another Scala generator.

Let me know what you guys (Scala developers) prefer.

Reply:
`scala.math.*` `BigInt` and `BigDecimal` are preferable types in Scala. No import required for that types.

Reply:
Done via https://github.com/OpenAPITools/openapi-generator/pull/7876/commits/bd3a773b3fa968a7a6878409d317323ef49e6dbb. Please have another look when you've time. Thanks.